src/vma/main.cpp

```diff
@@ -1283,9 +1285,11 @@ void get_env_params()
 	if ((env_ptr = getenv(SYS_VAR_TIMER_RESOLUTION_MSEC)) != NULL)
 			safe_mce_sys().timer_resolution_msec = atoi(env_ptr);
 
-	if ((env_ptr = getenv(SYS_VAR_TCP_TIMER_RESOLUTION_MSEC)) != NULL) {
+	if ((env_ptr = getenv(SYS_VAR_TCP_TIMER_RESOLUTION_MSEC)) != NULL)
 			safe_mce_sys().tcp_timer_resolution_msec = atoi(env_ptr);
-	}
+
+	if ((env_ptr = getenv(SYS_VAR_INTERNAL_THREAD_TCP_TIMER_HANDLING)) != NULL)
+			safe_mce_sys().internal_thread_tcp_timer_handling = (internal_thread_tcp_timer_handling_t)atoi(env_ptr);
 
```

Comment:
We should check for the validity of values assigned into internal_thread_tcp_timer_handling (enum type).
Using strtol() is better by atoi() for the reason you have mentioned but -
1. Instead of replacing just one line there is a need to replace all occurrences of atoi. That deserves a PR of its own.
2. Even when using strtol need to check validity of internal_thread_tcp_timer_handling assignment. 

I will add validity check while still using atoi()